src/issuance.rs

```diff
@@ -653,6 +671,54 @@ mod tests {
         )
     }
 
+    fn generate_identity_point_asset_base() -> AssetBase {
+        let identity_point =
+            (pallas::Point::generator() * -pallas::Scalar::one()) + pallas::Point::generator();
```

Comment:
What is the source of this expression?

Reply:
Avoid `pallas::` by importing the concrete expressions.

Reply:
The expression is from `pasta-curves-0.5.1/src/pallas.rs`, line 95 (GitHub permalink [here](https://github.com/zcash/pasta_curves/blob/7e3fc6a4919f6462a32b79dd226cb2587b7961eb/src/pallas.rs#L95)).

I used `generator()` to get a point on the curve instead of using the hard-coded `r` value over there, as any point on the should generate the identity this way.

Reply:
imported `pallas::Point` and `pallas::Scalar`